monai/transforms/croppad/functional.py

```diff
@@ -29,20 +31,34 @@
 
 
 def _np_pad(img: torch.Tensor, pad_width: list[tuple[int, int]], mode: str, **kwargs) -> torch.Tensor:
-    img_np = img.detach().cpu().numpy() if isinstance(img, torch.Tensor) else img
+    if isinstance(img, torch.Tensor):
+        if img.is_cuda:
+            warnings.warn(f"Padding: moving img {img.shape} on cuda to cpu for dtype={img.dtype} mode={mode}.")
+        img_np = img.detach().cpu().numpy()
+    else:
+        img_np = img
     mode = convert_pad_mode(dst=img_np, mode=mode).value
     if mode == "constant" and "value" in kwargs:
-        kwargs["constant_values"] = kwargs.pop("value")
-    out = torch.as_tensor(np.pad(img, pad_width, mode=mode, **kwargs))  # type: ignore
+        _kwargs = kwargs.copy()
```

Comment:
May I know why copy the `kwargs` here? Seems we can modify and use `kwargs` directly?

Thanks.